cli/config/version.go

```diff
@@ -33,45 +34,56 @@ Server: Not Configured`, false
 Server: Failed to get the server version - %s`, err.Error()), false
 	}
 
-	versionMatch := false
 	version := meta.GetVersion()
-	if !isSemver(version) || !isSemver(Version) {
-		// if either version is not semver, we can't compare them
-		// do a basic strict compare
-
-		versionMatch = version == Version
-	} else {
-
-		serverVersion, err := semver.NewVersion(version)
-		if err != nil {
-			return result + fmt.Sprintf(`
-Server: Failed to parse the server version - %s`, err.Error()), false
-		}
-
-		cliVersion, err := semver.NewVersion(Version)
-		if err != nil {
-			return result + fmt.Sprintf(`
-Failed to parse the CLI version - %s`, err.Error()), false
-		}
-
-		versionConstrait, err := semver.NewConstraint(fmt.Sprintf(">=%d.%d", cliVersion.Major(), cliVersion.Minor()))
-		if err != nil {
-			return result + fmt.Sprintf(`
-Failed to parse the CLI version constraint - %s`, err.Error()), false
-		}
-
-		versionMatch = versionConstrait.Check(serverVersion)
```

Comment:
I didn't change this logic (if you see any change, block the PR). I only detect if the CLI version if smaller than the server version and if so, show the outdated warning